viewmaster/viewmaster.py

```
@@ -260,128 +303,105 @@ def validate_holdings_paths(abspaths):
+    """
 
-        if os.path.islink(symlink):         # exists and is a symlink
-            try:
-                os.remove(symlink)
-            except OSError:
-                LOGGER.error('Cannot remove outdated symlink', symlink)
+    try:
+        paths = get_holdings_paths()
+        paths = validate_holdings_paths(paths, logger)
+    except Exception as e:
+        logger.exception(e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)